ShopifySharp/Services/FulfillmentService/FulfillmentServiceService.cs

```diff
@@ -0,0 +1,106 @@
+﻿using Newtonsoft.Json.Linq;
+using System.Net.Http;
+using ShopifySharp.Filters;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using ShopifySharp.Infrastructure;
+
+namespace ShopifySharp
+{
+    /// <summary>
+    /// A service for manipulating Shopify fulfillments.
+    /// </summary>
+    public class FulfillmentServiceService : ShopifyService
```

Comment:
Just curious, I haven't looked at the docs yet. Does this endpoint support counting the Fulfillment Services?

Reply:
No, there's no counting documented. I'm assuming most stores don't have a ton of fulfillment services registered, so there wasn't a reason to implement a count function.